ISPN-13975: Update property realm note in Security realms doc (#10195)
infinispan/infinispan: documentation/src/main/asciidoc/topics/proc_creating_server_security_realms.adoc

. Use the `security-realms` element in the `security` configuration to contain create multiple security realms.
. Add a security realm with the `security-realm` element and give it a unique name with the `name` attribute.
+

> [Important]

Reviewer: [IMPORTANT]

Author: Of course, I do not specify a snippet for this type of admonition.

====

> Do not add any special characters, such as a hyphen (-), an amperand (&), and so on, to a security realm name. 

Reviewer 2: ```suggestion
Do not add special characters, such as hyphens (`-`) or ampersands (`&`), to security realm names. 
```

> Special characters in security realm names can make them unusable by {brandname} Server endpoints.

Reviewer 2: ```suggestion
{brandname} Server endpoints can become unreachable if security realm names contain special characters.
```

Author: Hi @oraNod . Implemented your suggestion, changed the `name: "trust-store-realm"` entry in the YAML file, and then updated the commit message.

> Specifying any special characters for a security realm name might make your security realm unusable by your {brandname} Server's endpoints.

Reviewer 2: ```suggestion
Special characters in security realm names can make them unusable by {brandname} Server endpoints.
```

. Use the `security-realms` element in the `security` configuration to contain create multiple security realms.
. Add a security realm with the `security-realm` element and give it a unique name with the `name` attribute.

> +

Author: Yes, good point. Add an important admonition under this step.

[discrete]
== Multiple property realms

You can define multiple security realms in an XML, a JSON, or a YAML file. The following examples demonstrate defining multiple security realms in each file type.

[NOTE]
====

> When defining the name of your security realm in the `name` field, ensure that you do not include any special characters in the name, such as a hyphen (-), an underscore) (_), and so on. By using special characters, you might cause issues with an authentication mechanism that uses the `REST` endpoint or cause connections issues with the {brandname} CLI.  

Reviewer: One sentence per line

[discrete]
== Multiple property realms

You can define multiple security realms in an XML, a JSON, or a YAML file. The following examples demonstrate defining multiple security realms in each file type.

[NOTE]

> ====

Reviewer: Good. So can we drop this note? Now the info is duplicated

[discrete]
== Multiple property realms

> You can define multiple security realms in an XML, a JSON, or a YAML file. The following examples demonstrate defining multiple security realms in each file type.

Reviewer: ```suggestion
The following configuration shows how you can configure multiple security realms in XML, JSON, or YAML format.
```

Reviewer: Or something similar. These are not file types

Author: Hi @dvagnero .

Thank you for the review comment. I updated the paragraph. WDYT, keep or remove the second sentence?

Author: Hi @dvagnero . Just a reminder on the above ^

Reviewer: I think you can shorten it and be more specific. Instead of different configuration formats you can say "in XML, JSON, and YAML file or format. One sentence is enough

Author: Thanks, @dvagnero . Updated.
